src/test/datascience/jupyterPasswordConnect.unit.test.ts

```diff
@@ -117,6 +117,7 @@ suite('JupyterPasswordConnect', () => {
     }
 
     test('getPasswordConnectionInfo', async () => {
+        when(appShell.showInputBox(anything())).thenReturn(Promise.resolve('Python'));
```

Comment:
Updated previous good password check to also actually check the password in the fetch.